bin/import_cv2.py

```diff
@@ -9,49 +9,42 @@
 
 import csv
 import sox
+import argparse
```

Comment:
General question that I'll just put here: What about other languages?

CV2 now has about 15 languages and if I want to import anything other than English, it doesn't look like we support that. Maybe another PR to add non-English support?

Reply:
One has to specify the according alphabet and it should be fine for most cases. I see the `normalize` option as something hacky that is really optional and only makes sense for English as of now. None of the other changes is specifically targeting English and this PR anyhow doesn't aim for solving all of these problems. It's purpose is to allow filtering samples with unacceptable characters. Common character conversion and filtering should be done in `validate_label` (like casing), while everything else is a matter of providing the right alphabet.